src/sentry/static/sentry/app/views/issueList/overview.jsx

```diff
@@ -680,33 +655,13 @@ const IssueListOverview = createReactClass({
     }
 
     const {params, organization, savedSearch, savedSearches} = this.props;
-    const {selectedProject} = this.state;
     const query = this.getQuery();
 
-    // If we have a selected project set release data up
-    // enabling stream actions
-    let hasReleases = false;
-    let projectId = null;
-    let latestRelease = null;
-
-    const projects = this.getGlobalSearchProjects();
-
-    if (selectedProject) {
-      hasReleases = new Set(selectedProject.features).has('releases');
-      latestRelease = selectedProject.latestRelease;
-      projectId = selectedProject.slug;
-    } else if (projects.length === 1) {
-      // If the user has filtered down to a single project
-      // we can hint the autocomplete/savedsearch picker with that.
-      projectId = projects[0].slug;
```

Comment:
Also, I'm not sure how exactly this is used. The comment implies that it should be passed to IssueListFilters to "hint the autocomplete/savedsearch picker" and it is passed to IssueListFilters, but not used within that component.

Reply:
Not sure either but if it's not used in `IssueListFilters` then seems fine.